modules/siddhi-core/src/main/java/org/wso2/siddhi/core/SiddhiAppRuntime.java

```diff
@@ -693,4 +693,24 @@ public void enableStats(boolean statsEnabled) {
             log.debug("Siddhi App '" + getName() + "' statistics reporting not changed!");
         }
     }
+
+    /**
+     * To enable and disable Siddhi App playback mode on runtime.
+     *
+     * @param playBackEnabled whether statistics is enabled or not
+     */
+    public void enablePlayBack(boolean playBackEnabled) {
+        this.siddhiAppContext.setPlayback(playBackEnabled);
+    }
+
+    /**
+     * To provide playback options on runtime.
+     *
+     * @param playBackOptions map containing optional parameters
+     */
+    public void setPlayBackOptions(Map<String, Long> playBackOptions) {
```

Comment:
get idleTime and IncrementInMilliseconds as two arguments along with enablePlayBack
ie enablePlayBack(playBackEnabled, idleTime, IncrementInMilliseconds);

Reply:
This is fixed